Trip Planner/trips_manage.php

```diff
@@ -140,7 +141,7 @@
         return $filters;
     });
 
-    $filters['showActive:Y'] = __m('Upcoming / Approved Trips');
+    $filters['showActive:Y'] = __('Upcoming / Approved Trips');
```

Comment:
This should be __m, as it's a custom module string.
```suggestion
    $filters['showActive:Y'] = __m('Upcoming / Approved Trips');
```